__tests__/app.test.js

```diff
@@ -454,3 +454,47 @@ test("POST: 404 with Not Found for non-existent article ID", () => {
       expect(response.body.msg).toBe("Bad Request")
     })
 })
+// 8==========
```

Comment:
You could use describe blocks to help with separating tests both in the file and the console when you run them 🙂